sdk/python/kfp/components/structures/__init__.py

```diff
@@ -0,0 +1 @@
+from .._structures import *
```

Comment:
Can we rename the current structures dir to schema, then we just rename _structures.py to structures.py. I think it'll be better to let the package path aligned with the dir layout.

Reply:
We might do this in future, but for now I'm a bit cautious to make the `kfp.components._structures` unavailable. If it's OK, I'd propose changing that in a later release.

Reply:
Sure.